src/main/steps/respondent/hub-page/latest-update-content/final-order-complete.njk

```diff
@@ -0,0 +1,4 @@
+{% if isFinalOrderComplete %}
+  <p class="govuk-body">{{ finalOrderComplete.line1 }}</p>
+  <p class="govuk-body">{{ finalOrderComplete.line2 | safe }}</p>
```

Comment:
same

Reply:
Moved the links to nunjucks files and renamed references